utils/dataloaders.py

```diff
@@ -159,14 +163,17 @@ def create_dataloader(path,
             image_weights=image_weights,
             prefix=prefix)
 
+    # Access the im_files attribute
+    image_files = dataset.im_files
```

Comment:
It's a good practice to declare the variables as close to their usage as possible. 

- The variable definition and any related comments are easy to find.
- The reader knows that this variable is used for the first time there.
- By declaring them close to their usage, you implicitly state this is where the variable is used.
- It might help with identifying the refactoring of code. For example, by adding the variables inside the scope of the condition above you might come to decide that actually the condition could be done elsewhere.